pkg/generator/generator.go

```diff
@@ -209,6 +202,29 @@ func cleanDestination(g *generator) error {
 	return nil
 }
 
+func processPath(g *generator, fileName string, parmsObject interface{}, overrides *tmpl.Overrides) (string, error) {
+	newFileName := strings.Replace(fileName, g.pathPrefix, "", 1)
+	newFileName = applyOverrides(newFileName, overrides)
+
+	tmpl, err := template.New("").Funcs(sprig.TxtFuncMap()).Funcs(createFuncMap()).Parse(newFileName)
+	if err != nil {
+		return "", err
+	}
+
+	buf := &bytes.Buffer{}
+	if err := tmpl.Execute(buf, parmsObject); err != nil {
+		return "", err
+	}
+
+	newFileName = buf.String()
+
+	if runtime.GOOS == "windows" {
+		newFileName = strings.ReplaceAll(newFileName, "/", `\`)
+	}
```

Comment:
Are we sure this is needed? I believe that when we write the file to the disk it should automatically convert the path when we're in the Windows runtime.

Reply:
If we don't do this, there is potential to have mixed path separators in the resulting string on Windows.  It may work to write the path like that.  However, the main reason i did this was because, later on, we use path functions to parse out portions of this path, and I didn't want to take the chance that they might get confused by intermingling "/" and "\".

I did just check though and it looks like the only function we use after this is set is `filepath.Base` which does handle both "/" and "\".  I tested it too without this and it did work, so I'll remove that.